include/wx/gtk/private/webkit.h

```diff
@@ -0,0 +1,57 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/gtk/private/webkit.h
+// Purpose:     wxWebKitGtk RAII wrappers declaration
+// Author:      Jose Lorenzo
+// Created:     2017-08-21
+// Copyright:   (c) 2017 Jose Lorenzo <josee.loren@gmail.com>
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_GTK_PRIVATE_WEBKIT_H_
+#define _WX_GTK_PRIVATE_WEBKIT_H_
+
+#include <webkit2/webkit2.h>
+
+// ----------------------------------------------------------------------------
+// Convenience class for WebKitJavascriptResult on scope exit automatically
+// ----------------------------------------------------------------------------
+
+class wxWebKitJavascriptResult
+{
+public:
+    explicit wxWebKitJavascriptResult(WebKitJavascriptResult *r) : m_jsresult(r) { }
+    ~wxWebKitJavascriptResult() { webkit_javascript_result_unref (m_jsresult); }
+
+    operator WebKitJavascriptResult *() const { return m_jsresult; }
+
+private:
+    WebKitJavascriptResult *m_jsresult;
+
+    wxDECLARE_NO_COPY_CLASS(wxWebKitJavascriptResult);
+};
+
+#include <JavaScriptCore/JSStringRef.h>
```

Comment:
Minor style: please include all headers near the top of the file, as customary, instead of sprinkling them over it.